### PR TITLE
[Feature] Upgrade Package Support To L8

### PR DESCRIPTION
**Pull request by dljfield**

### CODE REVIEW

File: phpunit.xml.dist.bak [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mauricius) [inline phpunit.xml.dist.bak:1]: I guess you can remove this file!

File: src/Models/Campaign.php [hunk 2]
```diff
@@ -300,7 +361,8 @@ public function canBeCancelled(): bool
         // we can cancel campaigns that still have draft messages, because they haven't been entirely dispatched
         // a campaign that doesn't have any more draft messages (i.e. they have all been sent) cannot be cancelled, because the campaign is completed
 
-        if ($this->status_id === CampaignStatus::STATUS_SENT && $this->save_as_draft && $this->sent_count !== $this->messages()->count()) {
+        if ($this->status_id === CampaignStatus::STATUS_SENT && $this->save_as_draft && $this->sent_count !== $this->messages(
```
Comment (mauricius) [inline src/Models/Campaign.php:364]: indentation issue

**Review by mauricius — APPROVED** [review]

File: phpunit.xml.dist.bak (hunk 1 above)
Comment (dljfield) [reply phpunit.xml.dist.bak]: Removed

File: src/Models/Campaign.php (hunk 2 above)
Comment (dljfield) [reply src/Models/Campaign.php]: Reformatted this long if into multiple lines so it's easier to read and avoids that weirdness with the autoformatting.

**Pull request merged**